### DISCUSSION TITLE
[#940477] Arrow does not stick with the cursor very well

### DISCUSSION THREAD

**Discussion by tryashtar** [Mojang/minecraft-editor#37] [Bugs]
When dragging the arrows to resize a selection, the arrow is not very responsive, and gets quickly descyned from your cursor
https://user-images.githubusercontent.com/26075577/223857495-05f839c9-8a29-4be6-9529-015fbed95094.mp4

**Comment by minecraftchloe** [comment]
I've noticed this, too -- updated title to reflect internal tracking ID

**Comment by minecraftchloe** [comment]
We've done some polish on this - how's it feeling now?

**Comment by tryashtar** [comment]
Here's what I think feels weird (in 1.20.50):
https://github.com/Mojang/minecraft-editor/assets/26075577/0c79b7a5-bca4-4981-992c-96f83e74d3f1
I move the mouse up just until the box changes size, then I immediately go back. Instead of reverting to the original size, the box stays larger and I have to move even further to shrink it. Same thing happens towards the bottom, I shake the mouse in the area I expect the box to change (because it just did), but it doesn't respond. Does that make sense?
